cinema/serializers.py

```diff
@@ -125,10 +124,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
+            ValidationError,
         )
         return data
```

Comment:
Ensure that the `validate_ticket` method in the `Ticket` model is implemented correctly. This method is used to validate the ticket details such as row, seat, and movie session. If this method is not implemented or has issues, it could lead to validation errors.